### ISSUE TITLE
Generated content provider does not compile (updateWithOnConflict)

### ISSUE THREAD

**Issue by ron-k** [xcesco/kripton#23]
Let DAO method annotated with the following:
 ```
@BindContentProviderEntry(path = "${pref.verticalId}")
    @BindSqlUpdate(conflictAlgorithm = ConflictAlgorithmType.IGNORE, where = "verticalId=${pref.verticalId}")
    int update(VerticalPref pref);

```
The combination of `conflictAlgorithm` and `where` in the annotation `@BindSqlUpdate` generates a code with compilation error, as following (note the extra '**)**' before ", 4"):

`int result = database().updateWithOnConflict("vertical_pref", contentValues.values(), sqlWhereStatement, _contentValues.whereArgsAsArray()), 4);`

**Comment by xcesco** [comment]
I'm working on the 6.x branch. I reproduce the bug with the following scenario:

https://github.com/xcesco/kripton/tree/v6.x/kripton-processor/src/test/java/sqlite/git23/bug

The fix will be released with 6.0.0-rc.7. I'm running the test and preparing the release.

**Comment by xcesco** [comment]
Version 6.0.0-RC.7 is just released with the fix. In a couple of hours will available on bintray and maven repositories
